app/javascript/components/message.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+const GET_RANDOM_MSG_REQUEST = 'GET_RANDOM_MSG_REQUEST';
+const GET_RANDOM_MSG_SUCCSESS = 'GET_RANDOM_MSG_SUCCSESS';
+
+const getRandomMsgSuccess = (json) => ({
+  type: GET_RANDOM_MSG_SUCCSESS,
+  json,
+});
+
+const getRandomMsg = () => (dispacth) => {
+  dispacth({ type: GET_RANDOM_MSG_REQUEST });
+  return fetch('api/v1/randomMessage')
+    .then((response) => response.json())
+    .then((json) => dispacth(getRandomMsgSuccess(json)))
+    .catch((error) => console.log(error));
+};
+
+export default function Message(props) {
```

Comment:
- [ ] It will be nice to also change the extension here. Same reason as above.

Reply:
- [ ] By extension, I mean rename the `message.jsx` to `message.js`.  React components are javascript functions and should only have the `.js` extensions. It will be nice to fix this.

Reply:
Hey @enaburekhan  Kindly do some research before asking the same _not accurate_ suggestions,

Reply:
My reason above also suffice here

Reply:
Hey @enaburekhan I don't understand what do you mean? 

>  By extension, I mean rename the message.jsx to message.js. React components are javascript functions and should only have the .js extensions. It will be nice to fix this. 

You need all react components to be in .jsx extenetion check this message from the terminal: 
![image](https://user-images.githubusercontent.com/29541335/150786289-8e0d36dd-71fd-4118-b692-889c55d2e8a4.png)